renderdoc/data/glsl/debuguniforms.h

```diff
@@ -89,7 +89,7 @@ struct Vec4u
 #ifndef OPENGL_ES
 #define PRECISION
 #else
-#define PRECISION mediump
+#define PRECISION highp
```

Comment:
I think this will not cause any problems, probably the calculations will be more accurate.